sdk/tests/tutorials/ibor/test_portfolio_group.py

```diff
@@ -0,0 +1,260 @@
+            code="portfolio_group-{}".format(sub_guid),
+            display_name="portfolio_group-{0}".format(sub_guid),
+            values=[],
+            created=effective_date
+        )
+
+        # create the sub portfolio group in LUSID in the specified scope
+        sub_portfolio_group_result = self.portfolio_group_api.create_portfolio_group(
+            scope=TestDataUtilities.tutorials_scope,
+            create_portfolio_group_request=sub_portfolio_group_request
+        )
+
+        # create sub portfolio group
+        add_subgroup_to_group = self.portfolio_group_api.add_sub_group_to_group(
+            scope=TestDataUtilities.tutorials_scope,
+            code=portfolio_group_request.code,
+            resource_id=models.ResourceId(
+                scope=TestDataUtilities.tutorials_scope,
+                code=sub_portfolio_group_request.code
+            ),
+            effective_at=effective_date
+        )
+
+        # assertions
+        self.assertEqual(add_subgroup_to_group.sub_groups[0].code,sub_portfolio_group_request.code)
+
+    @lusid_feature("F8-4")
+    def test_delete_portfolio_from_group(self):
+
+        guid = str(uuid.uuid4())
+        effective_date = datetime(2018, 1, 1, tzinfo=pytz.utc)
+
+        # create a portfolio and add it to the group
+        # create portfolio
+        # set up the portfolio
+        portfolio_request = models.CreateTransactionPortfolioRequest(
+            display_name="portfolio-{0}".format(guid),
+            code="id-{0}".format(guid),
+            base_currency="GBP",
+            created=effective_date
+        )
+
+        # create the portfolio in LUSID in the specified scope
+        portfolio_result = self.transaction_portfolios_api.create_portfolio(
+            scope=TestDataUtilities.tutorials_scope,
+            create_transaction_portfolio_request=portfolio_request
+        )
+
+        # create portfolio group
+        portfolio_group_request = models.CreatePortfolioGroupRequest(
+            code="portfolio_group-{}".format(guid),
+            display_name="portfolio_group-{0}".format(guid),
+            values=[],
+            created=effective_date
+        )
+
+        # create the portfolio group in LUSID in the specified scope
+        portfolio_group_result = self.portfolio_group_api.create_portfolio_group(
+            scope=TestDataUtilities.tutorials_scope,
+            create_portfolio_group_request=portfolio_group_request
+        )
+
+        # add transactions portfolio to group
+        add_portfolio_request = self.portfolio_group_api.add_portfolio_to_group(
+            scope=TestDataUtilities.tutorials_scope,
+            code=portfolio_group_result.id.code,
+            resource_id=models.ResourceId(
+                scope=TestDataUtilities.tutorials_scope,
+                code=portfolio_result.id.code
+            ),
+            effective_at=effective_date
+        )
+
+        # remove the portfolio from our group
```

Comment:
Again, perhaps remove lines 154 to here? You can re-use the same portfolio groups as above?